plugins/CoreConsole/Commands/ComputeJsAssetSize.php

```diff
@@ -19,9 +20,18 @@
 use Piwik\SettingsPiwik;
 use Piwik\Theme;
 
+/**
+ * @phpstan-type TOTALS array{merged: int, gzip: int}
```

Comment:
Do we have a convention on how to name pseudo-types like this (e.g. all capitals, or T_Something, or ClassLikeName etc.) or is it PHPStan recommended way? With all caps it feels more like a constant.